.ci/openshift-ci/run_smoke_test.sh

```diff
@@ -56,7 +56,11 @@ else
     port=$(oc get service/http-server-service -o jsonpath='{.spec.ports[0].nodePort}')
 fi
 
-curl "${host}:${port}${hello_file}" -s -o hello_msg.txt --retry 60 --retry-delay 1 --retry-all-errors
+rm -f hello_msg.txt
+for i in $(seq 60); do
```

Comment:
hi @ldoktor , good catch.

One suggestion is to use the `waitForProcess` (e.g. `waitForProcess $wait_time $sleep_time "$cmd"`) utility.

Reply:
Sure